mapeditor/mapcontroller.cpp

```diff
@@ -613,16 +613,49 @@ ModCompatibilityInfo MapController::modAssessmentAll()
 ModCompatibilityInfo MapController::modAssessmentMap(const CMap & map)
 {
 	ModCompatibilityInfo result;
+
+	auto extractEntityMod = [&result](const auto & entity) 
+	{
+		auto modName = QString::fromStdString(entity->getJsonKey()).split(":").at(0).toStdString();
+		if(modName != "core")
+			result[modName] = VLC->modh->getModInfo(modName).getVerificationInfo();
+	};
+
```

Comment:
Perhaps instead we should add `getModScope` method to entity class?
More clear and `getJsonKey()` is often implemented as `return modScope + ':' + identifier;`